_episodes/09-debugging.md

```diff
@@ -140,7 +140,7 @@ we want to localize the failure to the smallest possible region of code:
     then concentrate on either the first or second half,
     and so on.
 
-2.  N things can interact in N! different ways,
+2.  N things can interact in N<sup>2</sup> different ways,
```

Comment:
Something went wrong... Could you please fix this PR... or submit a new one?

Reply:
Odd. Not sure what happened, so I'll submit a new PR as you suggest.